src/pages/docs/step-ca/configuration.mdx

```diff
@@ -624,103 +847,111 @@ To configure an existing CA for Azure Key Vault, or to import an existing Azure
-✔ SSH User Private Key: awskms:key-id=cf28e942-1e10-4a08-b84c-5359af1b5f12
-✔ SSH Host Public Key: ssh_host_ca_key.pub
-✔ SSH Host Private Key: awskms:key-id=cf28e942-1e10-4a08-b84c-5359af1b5f12`}</CodeBlock>
+Great. Next, we'll repeat the process for the Intermediate CA:
 
-The `--region` parameter is only required if your AWS configuration does not define a region.
+```shell nocopy
+$ step kms create --json --kms 'yubikey:' 'yubikey:slot-id=9c'
+$ step certificate create --profile intermediate-ca \
+   --kms 'yubikey:pin-value=123456' \
+   --ca root_ca.crt \
+   --ca-key 'yubikey:slot-id=9a' \
+   --key 'yubikey:slot-id=9c' \
+   "Smallstep Intermediate CA" intermediate_ca.crt
+```
 
-Run the `step-awskms-init --help` command for more options.
+Here, the `--ca-key` is the root CA key id; the `--key` is the intermediate CA key id.
 
-### YubiKey PIV
+Output:
 
-<Alert severity="info">
-  <div>
-    <strong>This feature is not enabled by default</strong><br />
-    To enable support, you will need to build <code>step-ca</code> yourself, using CGO support.<br />
-    See <a href="https://github.com/smallstep/certificates/blob/master/docs/CONTRIBUTING.md#build-step-ca-using-cgo">Building From Source Using CGO</a>. 
-  </div>
-</Alert>
+```
+Your certificate has been saved in intermediate_ca.crt.
+```
 
-You can leverage a hardware [YubiKey](https://www.yubico.com/)—and the [YubiKey PIV application](https://www.yubico.com/authentication-standards/smart-card/)—to store your CA keys and sign TLS certificates.
+Now you should have both `root_ca.crt` and `intermediate_ca.crt` certificate PEM files.
+You'll need these files for your CA configuration, below.
 
-#### Prerequisites and Caveats
-* To enable YubiKey support in `step-ca`, you must follow our [Instructions for building from source using CGO](https://github.com/smallstep/certificates/blob/master/docs/CONTRIBUTING.md#build-step-ca-using-cgo)
-* You will need a YubiKey that supports the PIV application: YubiKey 5 NFC, YubiKey 5 Nano, YubiKey 5C, or YubiKey 5C Nano
-* [Certificate slots](https://developers.yubico.com/PIV/Introduction/Certificate_slots.html) 9a, 9c, 9d, 9e, and 82-95 are supported
-* You can use the YubiKey for X.509 and SSH CAs; however, the `step-yubikey-init` utility only supports X.509 at this time. For SSH support, you must manually generate and configure SSH CA keys on the YubiKey.
+Optionally, for safekeeping, you may wish to import the certificates into the YubiKey. To do this, you'll need Yubico's [`ykman` CLI utility](https://developers.yubico.com/yubikey-manager/). Run:
 
-The initialization of the public key infrastructure (PKI) for YubiKeys is not currently integrated into `step`, but an experimental tool named `step-yubikey-init` addresses this use case. In a future release, this tool will be integrated into `step`.
+```shell nocopy
+$ ykman piv certificates import 9a root_ca.crt
+$ ykman piv certificates import 9c intermediate_ca.crt
```

Comment:
Hmm I tried this, and I got the following:

```bash
$ step kms certificate --import root_ca.crt --kms 'yubikey:pin-value=123456' 'yubikey:slot-id=9a'
Error: error opening yubikey: connecting to smart card: the smart card cannot be accessed because of other connections outstanding
exit status 1
```

I'm able to import the cert with `ykman` and I'm able to run `step kms create` as well, so it's just this command that's failing.

Reply:
It happens to me too, it should be a bug with the YubiKey implementation, it used to work with PKCS#11

Reply:
Ok, I just created https://github.com/smallstep/step-kms-plugin/issues/5 for this.